src/Serializer/AbstractItemNormalizer.php

```diff
@@ -280,6 +280,9 @@ private function denormalizeRelation(string $attributeName, PropertyMetadata $pr
             } catch (ItemNotFoundException $e) {
                 throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
             } catch (InvalidArgumentException $e) {
+                if ('' === $value && ($propertyMetadata->isWritableLink() || $this->resourceClassResolver->isResourceClass($className))) {
+                    return null;
+                }
                 // Give a chance to other normalizers (e.g.: DateTimeNormalizer)
```

Comment:
@dunglas here is this place. We normalizing relation, we got string, so it must definitely be IRI and no any chances for other. I've just tried to fix the symptom for exception didn't crush the application